Automattic/wp-calypso
# Help Center: Fix chat feedback msg

| Before  | After |
| ------------- | ------------- |
|  <img width="774" alt="Screenshot 2024-11-22 at 16 30 23" src="https://github.com/user-attachments/assets/41af308a-b8ae-43eb-9d7e-ef7345d0edae"> |  <img width="698" alt="Screenshot 2024-11-22 at 16 29 45" src="https://github.com/user-attachments/assets/933c0eee-7ea4-4a2d-90fd-5be4fad2eea4"> |

Fixes https://github.com/Automattic/dotcom-forge/issues/9929

## Proposed Changes

* Show conversation feedback message and link

## Why are these changes being made?


* The survey link was not shown, only the message content

## Testing Instructions



* Checkout this PR
* Use the `help-center-experience` flag
* Open a closed/finished conversation
* The feedback message and link should be displayed
